YouTube.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# Depends upon:
+#  youtube-dl
+#  aria2c
```

Comment:
Actually the package is just "aria2" even though the downloader's full name is "aria2c". You might want to change this.